src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { PeopleTable } from '../components/PeopleTable/PeopleTable';
+import { PeopleFilters } from '../components/PeopleFilters/PeopleFilters';
+import { useSearchParams } from 'react-router-dom';
+import { getFilteredPeople } from '../utils/getFiltredPeople';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(false);
+  const [searchParams] = useSearchParams();
+
+  const hasError = !isLoading && error;
+  const noPeople = !isLoading && people.length === 0;
+  const showContent = !isLoading && people.length > 0;
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') || '';
+  const centuries = searchParams.getAll('centuries') || [];
```

Comment:
The `centuries` parameter should not default to an empty array when using `searchParams.getAll()`, as it already returns an empty array if the parameter is not present. This default assignment is redundant.